tests/testthat/setup.R

```diff
@@ -0,0 +1,79 @@
+
+## SET UP ----
+
+  # Set teardown environment
+  teardownEnv <- if (testthat::is_testing()) testthat::teardown_env() else .GlobalEnv
+
+  # Set module path
+  testDirs <- list(
+    module = ifelse(testthat::is_testing(), dirname(dirname(getwd())), getwd())
+  )
+
+  # Source module code
+  suppressPackageStartupMessages(library(SpaDES.core))
```

Comment:
module code manually sourced here: is there a better way?

Reply:
Will be done by converting module to an R package, and by accessing event functions via an initiated `simList` object.